### PR TITLE
Automated README generation

### PR DESCRIPTION
**Pull request by dpatel257**
Not sure where to write 'how-to' for this script. Wiki ? OR somwhere in README?

it requires to have python3 on the host.
run it as 
'sudo python3 script/generate_readme.py'

### CODE REVIEW

File: QAT/centos-7.4/media/dev/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+|libvpx|BSD 3-clause "New" or "Revised" License|
+|Aomedia AV1 Codec Library|BSD 2-clause "Simplified" License|
+|x264|GNU General Public License v2.0 or later|
+|x265|GNU General Public License v2.0 or later|
+|dav1d|BSD 2-clause "Simplified" License|
+|Intel SVT-HEVC|BSD-2-Clause Plus Patent License|
+|Intel SVT-AV1|BSD-2-Clause Plus Patent License|
+|Intel SVT-VP9|BSD-2-Clause Plus Patent License|
+|gstreamer|GNU Lesser General Public License v2.1 or later|
+|gst orc|GNU Lesser General Public License v2.1 or later|
```
Comment (xwu2git) [inline QAT/centos-7.4/media/dev/README.md]: There is no GST in the nginx images.

Comment (jhou5) [reply QAT/centos-7.4/media/dev/README.md]: Not sure where to write 'how-to' for this script. Wiki ? OR somwhere in README?
     Add a cmake parameter UPDATE_IMAGE_README, setup "ON" to trigger the script. Then write the 'how-to' to cmake.md

**Comment by jhou5** [comment]
For questions - Not sure where to write 'how-to' for this script. Wiki ? OR somwhere in README?

Add a cmake parameter UPDATE_IMAGE_README, setup "ON" to trigger the script. Then write the 'how-to' to cmake.md

File: QAT/centos-7.4/media/dev/README.md (hunk 1 above)
Comment (jhou5) [reply QAT/centos-7.4/media/dev/README.md]: It is OK. GST is not included in nginx images.

**Comment by jhou5** [comment]
@dpatel257 , can you fix the conflicts?

File: CMakeLists.txt [hunk 2]
```diff
@@ -53,4 +61,6 @@ message(
     "-- Setting: DOCKER_PREFIX=${DOCKER_PREFIX}\n"
     "-- Setting: UPDATE_DOCKERFILES=${UPDATE_DOCKERFILES}\n"
     "-- Setting: UPDATE_IMAGELIST=${UPDATE_IMAGELIST}\n"
+    "-- Setting: UPDATE_IMAGE_README=${UPDATE_IMAGE_README}\n")
```
Comment (jhou5) [inline CMakeLists.txt]: ** Pls. remove Right bracket in this line. Error would be found when running cmake**
CMake Error: Error in cmake code at
/home/vcse/ovc/test_pr/Dockerfiles/CMakeLists.txt:65:
Parse error.  Expected a command name, got quoted argument with text "-- Setting: UPDATE_DOCKERHUB_README=${UPDATE_DOCKERHUB_README}\n".
-- Configuring incomplete, errors occurred!

**Pull request merged**